task1/src/main/java/com/fundacionjala/movies/Movie.java

```diff
@@ -0,0 +1,32 @@
+package com.fundacionjala.movies;
+
+public  class Movie {
+
+    public static final int CHILDRENS = 2;
+
+    public static final int REGULAR = 0;
+
+    public static final int NEW_RELEASE = 1;
+
+    private final String title;
+
+    private int priceCode;
+
+    public  Movie(String title, int priceCode) {
+        this.title = title;
+        this.priceCode = priceCode;
+    }
+
+    public int getPriceCode()
+    {
```

Comment:
Please Give format to all your classes
Take advantage from your IDE
Press ctrl + shift + alt + L